src/badge.html

```diff
@@ -1,4 +1,7 @@
-{{#> layout page="badge"}}
+{{#> layout page="badge"
+            title="Create badge - Livebook.dev"}}
```

Comment:
Usually it's the other way round, `Livebook.dev - Create badge` :)

Reply:
So I did a quick check here on all of my open tabs:

1. Zoho invoice, Gmail, GitHub, and Elixir Forum all do: "Create badge - Livebook.dev" style
2. Google Calendar does "Google Calendar April xyz"

So I would stick with the current approach. :)

Reply:
Ah right, I guess for Gmail/GitHub/forums it's the other way round because the subtitle is long (thread description, etc), but I don't have a strong preference, so let's keep as is :)